jax/lax/lax.py

```diff
@@ -435,11 +438,25 @@ def conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation=None,
     lhs_dilation = (1,) * (lhs.ndim - 2)
   if rhs_dilation is None:
     rhs_dilation = (1,) * (rhs.ndim - 2)
+  if batch_group_count != 1 and batch_group_count != lhs.shape[dimension_numbers[0][0]]:
+    # XLA currently doesn't support 1 < batch_group_count < batch_size 
```

Comment:
Can you leave a TODO here with a buganizer number so we can track when this is supported by XLA?

Reply:
I realized know that batch_group_count isn't as general as I though it was. If anyone really needs the performance we could add some logic to add batch_group_count in the transpose_rhs of a depthwise convolutions but it seems like it was never supposed to be used for grouped convolutions in general.